package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "todo-list",
+  "version": "1.0.0",
+  "description": "",
+  "private": "true",
+  "scripts": {
```

Comment:
"watch": "webpack --watch",
  "serve": "webpack serve --open",
   "build": "webpack",
   "start": "npm run serve & npm run watch"

Reply:
Please update scripts as in prev comment

Reply:
Done